src/pykeen/triples/triples_factory.py

```diff
@@ -228,6 +232,8 @@ def create(
         mapped_triples: MappedTriples,
         num_entities: Optional[int] = None,
         num_relations: Optional[int] = None,
+        entity_ids: Collection[int] = None,
+        relation_ids: Collection[int] = None,
```

Comment:
Is there any use case where they are not `None`? I am not sure whether all code would correctly work with custom, non-consecutive IDs

Reply:
Nope but I don't want to make that judgment in this PR since it would also mean we need to delete the code about num_entities and num_relations